deep_speech_2/model.py

```diff
@@ -118,6 +119,24 @@ def event_handler(event):
             num_passes=num_passes,
             feeding=feeding_dict)
 
+    def infer_loss_batch(self, infer_data):
```

Comment:
No big difference between loss and cost. I prefer loss for it is more commonly used.